tests/test_ziti_model.cpp

```diff
@@ -32,6 +32,29 @@ limitations under the License.
 
 using Catch::Matchers::Equals;
 
+TEST_CASE("posture response response", "[model]") {
```

Comment:
could be worth testing malformed json to ensure no crashy crash happens... also might be worthwhile to return empty values etc in there for the same reasons. or not ya know, just an idea ;)

Reply:
might be worth testing the values for boundary / crazy values too... timeoutRemaining of max int etc. again - just some thoughts

Reply:
That is outside the scope of what I added does. That would be the responsibility of the model framework and jsmn.